diff --git a/sites/all/modules/quicktabs/quicktabs.module b/sites/all/modules/quicktabs/quicktabs.module
index 3503c35..7d68ae1 100644
--- a/sites/all/modules/quicktabs/quicktabs.module
+++ b/sites/all/modules/quicktabs/quicktabs.module
@@ -193,11 +193,27 @@ function quicktabs_render($quicktabs) {
     }
   }
 
-  $tabs_count = count($quicktabs['tabs']);
+
+
+  //REPLACED with tab count that discounts index 
+  //so empty block does not display	
+  //$tabs_count = count($quicktabs['tabs']);
+  //if ($tabs_count <= 0) {
+  //  return '';
+  // }
+
+  $tabs_count = 0;
+  foreach($quicktabs['tabs'] as $key=>$value) {
+    if($value) {
+       $tabs_count++;
+    }
+  }
+
   if ($tabs_count <= 0) {
     return '';
   }
 
+
   if ($quicktabs['style'] == 'default') {
     $quicktabs['style'] = variable_get('quicktabs_tabstyle', 'nostyle');
   }
